js/src/main/scala/scalajson/ast/JValue.scala

```diff
@@ -95,7 +95,7 @@ object JNumber {
   *
   * @author Matthew de Detrich
   */
-final class JNumber(private[ast] val underlying: String) extends JValue {
+final class JNumber private[ast] (val underlying: String) extends JValue {
   @inline def value: String = underlying
```

Comment:
IMO, with the constructor parameter being a public field, this construct is now completely superfluous.
Or is there some benefit that I am not seeing?